templates/dynamic/results.html

```diff
@@ -41,7 +41,7 @@ <h2 class="sr-only">TaxBrain</h2>
 
     <div class="result-header">
       <div class="result-header-control">
-          <h1>{% flatblock "dynamic_results_header" %} {{ tables.result_years.0 }} - {{ tables.result_years | last }} </h1>
+          <h1>{% flatblock "taxbrain_pe_results_header" %} {{ tables.result_years.0 }} - {{ tables.result_years | last }} </h1>
```

Comment:
Ahh, that makes sense. So the header name change in `taxbrain/results.html` is correct?